server/reports/commission/commission-endpoint.js

```diff
@@ -0,0 +1,110 @@
+//module "commission-report"
```

Comment:
While working on this feature, we intentionally renamed commission-report to commission-endpoint. It's fine that the "module" comment here is out of date, but what really bugs me is that git is showing this as _all new_ code instead of as a diff from the original, renamed file. I do want to learn the better way to get diffs like that, if anyone knows them.
It might help git if we did one single commit that renamed (git mv) the file and then later commits where we change the file, but I think we did basically that and git is still showing us mediocre diffs here.